src/vscodeUtils.ts

```diff
@@ -17,10 +20,36 @@ export async function getContentFromFilesystem(uri: Uri) {
   }
 }
 
-export function shouldIncludeFile(path: string, config: ResolvedConfig) {
-  const { include, exclude } = config
-  return (
-    include.some(x => minimatch(path, x))
-    && exclude.every(x => !minimatch(path, x, { dot: true }))
-  )
+/**
+ * @see https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/node/workspace.ts
+ */
+export async function shouldIncludeFile(path: string, config: ResolvedConfig): Promise<boolean> {
+  const relativeId = relative(config.dir || config.root, path)
+  if (micromatch.isMatch(relativeId, config.exclude))
+    return false
+
+  if (micromatch.isMatch(relativeId, config.include))
+    return true
+
+  if (config.includeSource?.length && micromatch.isMatch(relativeId, config.includeSource)) {
```

Comment:
I don't think source tests are supported yet (#17), let's remove this check for now

Reply:
But actually, having this check here, it would be solved. So why not? The test would display in the test explorer, which would lead to it being runnable. If you want, I can check if it is supported?

Reply:
Ah okay. It seemed to work when opening a new insource test file, but I think that is just on the tab open event. I'll definitely remove it then.